app/views/languages/show.html.haml

```diff
@@ -0,0 +1,15 @@
+- locales = Rails.application.config.i18n.available_locales.map(&:to_s)
+= render partial: 'shared/header'
+
+.container
+  = form_with url: change_language_path(cb: @callback, r_id: @response_id), method: 'post' do
+    .row
+      .col.s12
+        - locales.each do |locale|
+          %p
+            %label
+              %input{name: 'locale', type: 'radio', value: locale, checked: @locale == locale}
+              %span=locale
+    .row
+      .col.s12
+        = button_tag('Submit', type: 'submit', class: 'btn waves-effect waves-light')
```

Comment:
Can we i18nnify this "Submit" prompt? I think there's probably a translation already but the word in Dutch would be "Opslaan"

Reply:
Oops, i missed it, thanks